digitalpublicgoods/ottaa-project.json

```diff
@@ -0,0 +1,107 @@
+{
+  "name": "OTTAA Project",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://ottaa-project.github.io/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "Yes",
+    "isSoftwarePltIndependent": "No",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://ottaa-project.github.io/docs/welcome",
+      "https://ottaa-project.github.io/PictogramsPredictionsLibrary/docs/readme/",
+      "Technology standards - https://ottaa-project.github.io/docs/techno",
+      "Libraries used - https://ottaa-project.github.io/docs/Resources",
+      "Code of conduct - https://ottaa-project.github.io/docs/codeofconduct"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "Data can be transfered using JSON format. Reference : https://ottaa-project.github.io/PictogramsPredictionsLibrary/docs/howtouse"
```

Comment:
The link here might also be outdated, couldn't find the exact replacement but this might work for the same purpose -> https://ottaa-project.github.io/docs/techno